client/src/App.tsx

```diff
@@ -63,6 +65,10 @@ const router = createBrowserRouter([
         path: "profile/:username",
         element: <Profile />,
       },
+      // {
+      //   path: "challenges/filter",
```

Comment:
Try not to put commented out code in a PR unless it will be necessary to uncomment it in the future.